app/views/api/guide/show.md

```diff
@@ -0,0 +1,43 @@
+---
+page_title: Register API Guidance
+title: Register API Guidance
+---
+
+Welcome to the Register API! This API allows you to access information about trainees and provides endpoints to update and create trainee data. 
+
+Below are the available endpoints.
+
+## Endpoints
+
+### GET /info
```

Comment:
Should the reference info be generated from say .yml specifications like Apply does to automate the formatting? (Not necessarily in this PR).

Reply:
I'm unsure how complex our documentation will be - I wonder if a simple markdown file for each version may suffice for now. can look at implementing the Apply method as soon as things become more complex though 👍